app/views/content/train-to-be-a-teacher/teacher-degree-apprenticeships.md

```diff
@@ -33,43 +45,16 @@ Non-maintained schools (for example, academies, independent schools and private
 
 You do not have to pay tuition fees, but you will not be eligible for student finance. 
 
-The teacher training providers confirmed so far will offer TDAs in secondary maths.  
-
-The providers offering teacher degree apprenticeships are:
-
-* Nottingham Trent University
-* Staffordshire University, in partnership with the Stoke-on-Trent and Staffordshire Teacher Education Collective
-* University College London
-* University of Brighton
-* University of Huddersfield
-* University of Nottingham
-* University of Wolverhampton
-* Xavier Teach Southeast, in partnership with the University of Sussex
-* University of Derby
-
-[View all courses for the TDA in secondary maths](https://find-teacher-training-courses.service.gov.uk).
-
 ## Who’s eligible to apply for a teacher degree apprenticeship?
-Teacher degree apprenticeships will be open to anyone who would like to gain a degree and become a qualified teacher. This includes, but is not limited to, teaching assistants, school leavers or people looking to change careers.
+Teacher degree apprenticeships are open for candidates to gain a degree and become a qualified teacher. This includes, but is not limited to, teaching assistants, school leavers or people looking to change careers.
```

Comment:
We hardly ever refer to someone as a `candidate` on GIT - do we need to here? Is it a bit confusing?

Reply:
Originally it said 'anyone' but then policy asked for it to be changed to 'candidate', shall see if we can change it back